src/software/thunderscope/thunderscope_config.py

```diff
@@ -11,33 +11,30 @@
 )
 import pyqtgraph
 import signal
-import qdarktheme
 import os
-from qt_material import apply_stylesheet, list_themes
+import qdarktheme
 
 
 class TScopeConfig:
```

Comment:
nit: this class design is kinda weird because it uses class variables and instance variables. It still works because python makes a copy of the instance variables when the class gets instantiated. it probably should only use instance variables.